### PR TITLE
Simplifications to Python code.

### PR DESCRIPTION
**Pull request by ejbaran**
Reduced read from file function to one line. Added missing imports. Removed unnecessary imports.

### CODE REVIEW

File: examples/smart_contracts/v2/python/account_delegation.py [hunk 1]
```diff
@@ -20,61 +19,28 @@ def wait_for_confirmation(client, txid):
...
     algod_client = algod.AlgodClient(algod_token, algod_address)
 
     myprogram = "samplearg.teal"
-    # myprogram = "<filename>"
     # Read TEAL program
-    data = load_resource(myprogram)
-    source = data.decode('utf-8')
+    data = open(myprogram, 'r').read()
     # Compile TEAL program
-    # // This code is meant for learning purposes only
```
Comment (ryanRfox) [inline examples/smart_contracts/v2/python/account_delegation.py:50]: I prefer to keep the commented TEAL program inline. This one is short enough to read and understand here.

File: examples/smart_contracts/v2/python/dryrun_debugging.py [hunk 2]
```diff
@@ -41,53 +33,31 @@ def dryrun_debug(lstx, mysource):
...
     algod_address = "http://localhost:4001"
 
-    # algod_token = "<algod-token>"
-    # algod_address = "<algod-address:port>"
-    # receiver = "<receiver-address>"
     receiver = "ATTR6RUEHHBHXKUHT4GUOYWNBVDV2GJ5FHUWCSFZLHD55EVKZWOWSM7ABQ"
     algod_client = algod.AlgodClient(algod_token, algod_address)
 
     myprogram = "samplearg.teal"
-    # myprogram = "<filename>"
```
Comment (ryanRfox) [inline examples/smart_contracts/v2/python/dryrun_debugging.py:58]: again :)

File: examples/smart_contracts/v2/python/contract_account.py [hunk 3]
```diff
@@ -20,63 +18,30 @@ def wait_for_confirmation(client, txid):
...
     algod_client = algod.AlgodClient(algod_token, algod_address)
    
     myprogram = "samplearg.teal"
-    # myprogram = "<filename>"
     # Read TEAL program
-    data = load_resource(myprogram)
-    source = data.decode('utf-8')
+    data = open(myprogram, 'r').read()
     # Compile TEAL program
-    # // This code is meant for learning purposes only
```
Comment (ryanRfox) [inline examples/smart_contracts/v2/python/contract_account.py:52]: again :)

File: examples/smart_contracts/v2/python/account_delegation.py [hunk 4]
```diff
@@ -20,61 +19,28 @@ def wait_for_confirmation(client, txid):
...
-    # algod_address = "http://localhost:4001"
-    algod_token = "6b3a2ae3896f23be0a1f0cdd083b6d6d046fbeb594a3ce31f2963b717f74ad43"
-    algod_address = "http://127.0.0.1:54746"
-    # algod_token = "<algod-token>"
-    # algod_address = "<algod-address:port>" 
-    # receiver = "<receiver-address>" 
+    algod_token = "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa"
+    algod_address = "http://localhost:4001"
+
     receiver = "ATTR6RUEHHBHXKUHT4GUOYWNBVDV2GJ5FHUWCSFZLHD55EVKZWOWSM7ABQ"
```
Comment (ryanRfox) [inline examples/smart_contracts/v2/python/account_delegation.py:27]: Is this TestNet faucet?

**Review by ryanRfox — CHANGES_REQUESTED** [review]
A couple things to consider in my comments. I'm willing to approve without changes.

File: examples/smart_contracts/v2/python/account_delegation.py (hunk 4 above)
Comment (ejbaran) [reply examples/smart_contracts/v2/python/account_delegation.py:27]: I don't know. It was just in the code already. I did not change.

File: examples/smart_contracts/v2/python/account_delegation.py (hunk 1 above)
Comment (ejbaran) [reply examples/smart_contracts/v2/python/account_delegation.py:50]: I find it redundant and distracting given the file is easily accessible in the same directory and referenced here. But I know this is subjective and I don't feel that strongly about it. I'll add it back in.

Comment (ejbaran) [reply examples/smart_contracts/v2/python/account_delegation.py:50]: Another method would be to just add the code as a string assigned to a variable and not reference the separate file at all.

**Review by ryanRfox — DISMISSED** [review]

**Review by JasonWeathersby — APPROVED** [review]

**Pull request merged**